sunpy/map/__init__.py

```diff
@@ -3,7 +3,6 @@
 
 isort:skip_file
 """
-from ._units import maxwell
```

Comment:
Should we leave an import of maxwell from astropy here in case anyone is using it from the `sunpy.map` namespace? If not this removal definitely needs a changelog entry.

Reply:
I hope we never advertised that it was in this namespace, units should only be used from the `u` space. This unit was added here to register it with astropy.

I don't have a good suggestion.

I forgot to add a removal changelog?! I will fix that.